metrics/metrics.go

```diff
@@ -0,0 +1,25 @@
+package metrics
```

Comment:
Don't call this package `metrics`, as that will force collisions with other packages we use.  Instead, prefix the name so we don't need to alias it, e.g. `webpametrics`